wondrous-app/components/Common/SubmissionStatus/constants.tsx

```diff
@@ -0,0 +1,6 @@
+export const VARIATIONS = {
```

Comment:
better to use `enum`

Reply:
Can you elaborate?

Reply:
Enums allow a developer to define a set of named constants. Using enums can make it easier to document intent, or create a set of distinct cases. TypeScript provides both numeric and string-based enums.

https://www.typescriptlang.org/docs/handbook/enums.html

in your case, it will be 
```
export enum Variations {
  Default = 'default',
  Rounded = 'rounded',
}

```